csharp/method.go

```diff
@@ -162,6 +162,7 @@ func (self *MethodDeclaration) WriteCode(writer CodeWriter) {
 		writer.Write(")")
 
 		if self.hasBase && self.base != nil {
+			writer.Write(" : ")
```

Comment:
what is "base" on a method declaration?

Reply:
i just moved the colon literal from one place to the other